src/index.ts

```diff
@@ -166,7 +166,7 @@ export function apply(ctx: Context, config: Config) {
         }
 
         Object.assign(parameters, {
-          image: image[1],
+          image,
```

Comment:
Is it no longer needed here? It would be better left to the `data` callback below.